ext/web/06_streams.js

```diff
@@ -9,6 +9,7 @@
 const core = globalThis.Deno.core;
 const ops = core.ops;
 import * as webidl from "ext:deno_webidl/00_webidl.js";
+import { structuredClone } from "ext:deno_web/02_structured_clone.js";
```

Comment:
Are we using `02_structured_clone` or `13_message_port` one?

https://github.com/denoland/deno/blob/72d9f0609094adee954b271db12ca4c834f6b860/ext/web/13_message_port.js#L332

Reply:
its fine to use 02